mapie/regression/time_series_regression.py

```diff
@@ -143,44 +141,94 @@ def fit(
         -------
         MapieTimeSeriesRegressor
             The model itself.
+
+        Raises
+        ------
+        ValueError
+            If the length of ``y`` is greater than
+            the length of the training set.
         """
-        self = super().fit(X=X, y=y, sample_weight=sample_weight)
-        self.conformity_scores_ = self._relative_conformity_scores(
+        check_is_fitted(self, self.fit_attributes)
+        X, y = cast(NDArray, X), cast(NDArray, y)
+        m, n = len(X), len(self.conformity_scores_)
+        if m > n:
+            raise ValueError(
+                "The number of observations to update is higher than the"
+                "number of training instances."
+            )
+        new_conformity_scores_ = self._relative_conformity_scores(
             X, y, ensemble=ensemble
         )
+        self.conformity_scores_ = np.roll(
+            self.conformity_scores_, -len(new_conformity_scores_)
+        )
+        self.conformity_scores_[
+            -len(new_conformity_scores_):
+        ] = new_conformity_scores_
         return self
 
-    def partial_fit(
+    def init_alpha(
+        self,
+        alpha: Optional[Union[float, Iterable[float]]] = None,
+        reset: bool = False
+    ) -> Optional[Union[float, Iterable[float]]]:
+        if 'current_alpha' not in self.__dict__ or reset:
+            self.current_alpha: dict[float, float] = {}
+
+        # ACI preprocessing (select virtual current alpha)
+        # This code snippet in the "aci" method ensures that when
+        # the same confidence level (alpha) is encountered more
+        # than once, it is mapped to a consistent value.
+        # This helps maintain reliability and predictability in
+        # the algorithm's computations specific to the "aci" method.
+        if alpha is not None:
+            alpha_np = cast(NDArray, check_alpha(alpha))
+            alpha_np = np.round(alpha_np, 2)
+            for ix, alpha_checked in enumerate(alpha_np):
+                alpha_np[ix] = self.current_alpha.setdefault(
+                    alpha_checked, alpha_checked
+                )
+            alpha = alpha_np
+        return alpha
+
+    def adapt_conformal_inference(
```

Comment:
```suggestion
    def _adapt_conformal_inference(
```

Reply:
Why not, but then we'd have to replace `partial_fit` with `_partial_fit`.

Reply:
In this case, I prefer to leave it as is.